extensions/amp-viewer-integration/0.1/test/integration/test-amp-viewer-integration.js

```diff
@@ -23,205 +23,210 @@ import {
 import {ViewerForTesting} from '../../viewer-for-testing';
 import {getSourceUrl} from '../../../../../src/url';
 
-describes.sandboxed('AmpViewerIntegration', {}, (env) => {
+describes.sandboxed('amp-viewer-integration', {}, (env) => {
   const ampDocSrc = '/test/fixtures/served/ampdoc-with-messaging.html';
-  // TODO(aghassemi): Investigate failure in beforeEach. #10974.
```

Comment:
Was there a failure in the `beforeEach`?

Reply:
Not really, not sure why this was skipped (issue #10974 doesn't offer much context).

However, while looking into this, I discovered that `'should handle unload correctly'` test was incorrect and got a chance to fix it & move it to the correct location since it is testing `AmpViewerIntegration`. So thanks for this comment!